app/Census/CensusColumnFatherBirthPlace.php

```diff
@@ -34,7 +34,7 @@ public function generate(Individual $individual, Individual $head = null) {
 		$father = $this->father($individual);
 
 		if ($father) {
-			return $this->notCountry($father->getBirthPlace());
+			return $this->notCountry($father->getBirthPlace()->getGedcomName());
```

Comment:
We now have `Place::lastPart()`.
For consistency, perhaps it is better to have `Place::notLastPart()` and remove `AbstractCensusColumn::notCountry()`?